### ISSUE TITLE
dkms rpm fails to install: "LINUX_VERSION_CODE" is not defined & "KERNEL_VERSION" is not defined

### ISSUE THREAD

**Issue by danielnorberg** [GoogleCloudPlatform/compute-virtual-ethernet-linux#57]
On a GCP instance running Oracle Linux 9 with kernel 6.9 ([uek-next](https://github.com/oracle/linux-uek/tree/ueknext/latest)):

```
$ cat /var/lib/dkms/gve/1.4.1/build/make.log
DKMS make.log for gve-1.4.1 for kernel 6.9.0-2.el9ueknext.x86_64 (x86_64)
Mon Jul  1 09:37:21 PM UTC 2024
make -C /lib/modules/6.9.0-2.el9ueknext.x86_64/build M=$PWD
make[1]: Entering directory '/usr/src/kernels/6.9.0-2.el9ueknext.x86_64'
warning: the compiler differs from the one used to build the kernel
  The kernel was built by: gcc (GCC) 13.2.1 20231205 (Red Hat 13.2.1-6)
  You are using:           gcc (GCC) 11.4.1 20231218 (Red Hat 11.4.1-3.0.1)
  CC [M]  /var/lib/dkms/gve/1.4.1/build/gve_main.o
  CC [M]  /var/lib/dkms/gve/1.4.1/build/gve_tx.o
  CC [M]  /var/lib/dkms/gve/1.4.1/build/gve_tx_dqo.o
  CC [M]  /var/lib/dkms/gve/1.4.1/build/gve_rx.o
In file included from /var/lib/dkms/gve/1.4.1/build/gve_rx.c:7:
/var/lib/dkms/gve/1.4.1/build/gve.h:12:6: error: "LINUX_VERSION_CODE" is not defined, evaluates to 0 [-Werror=undef]
   12 | #if (LINUX_VERSION_CODE >= KERNEL_VERSION(6,8,0))
      |      ^~~~~~~~~~~~~~~~~~
/var/lib/dkms/gve/1.4.1/build/gve.h:12:28: error: "KERNEL_VERSION" is not defined, evaluates to 0 [-Werror=undef]
   12 | #if (LINUX_VERSION_CODE >= KERNEL_VERSION(6,8,0))
```

**Comment by jfraker** [comment]
Hello, thanks for the report! This issue should be fixed in the next release, coming today or tomorrow.

**Comment by jfraker** [comment]
Thanks again for this, I did manage to get the release out today - so this should be fixed in v1.4.2. Please let me know if you run into any isuses.
